infrastructure/fargate-demo/README.md

```diff
@@ -0,0 +1,40 @@
+# Fargate demo
+
+Deploy a Hello World Node app as a container on ECS / Fargate.
+
+## Getting AWS credentials
+
+At time of writing our new AWS account isn't in gds-cli yet so we have to do a bit of fiddling to get CLI credentials.
+
+```bash
+aws-vault exec gds-users -- aws sts assume-role --role-arn arn:aws:iam::626456592666:role/FIRST_NAME.LAST_NAME-admin --role-session-name test --region eu-west-2 --serial-number arn:aws:iam::622626885786:mfa/FIRST_NAME.LAST_NAME@digital.cabinet-office.gov.uk --token-code YOUR_MFA_CODE_HERE
```

Comment:
Will give this a go shortly - does the id `626456592666` vary too?

Reply:
and the serial too?

Reply:
Nope, those are the account IDs of our AWS accounts - the role one is our prototype account (where the role's defined) and the serial one is a reference to your IAM user in the gds-users AWS account.

Reply:
OK ace I think I see what's going on here - looks like they're static